devbox.json

```diff
@@ -15,7 +15,11 @@
       "build": "find . -maxdepth 4 -type f -name go.mod -exec dirname {} \\; | xargs -I {} bash -c \"cd '{}' && echo build '{}' && go build -v ./...\"",
       "lint": "find . -maxdepth 4 -type f -name go.mod -exec dirname {} \\; | xargs -I {} bash -c \"cd '{}' && echo lint '{}' && golangci-lint run --fix --timeout 300s\"",
       "fmt": "find . -maxdepth 4 -type f -name go.mod -exec dirname {} \\; | xargs -I {} bash -c \"cd '{}' && echo fmt '{}' && go fmt ./...\"",
-      "test": "find . -maxdepth 4 -type f -name go.mod -exec dirname {} \\; | xargs -I {} bash -c \"cd '{}' && echo test '{}' && go test -race -cover -v ./...\""
+      "test": "find . -maxdepth 4 -type f -name go.mod -exec dirname {} \\; | xargs -I {} bash -c \"cd '{}' && echo test '{}' && go test -race -cover -v ./...\"",
+      "update-internal-deps": [
+        "find . -maxdepth 4 -type f -name go.mod -exec dirname {} \\; | xargs -I {} bash -c \"cd '{}' && echo update-internal-deps '{}' && go get -u go.jetpack.io/pkg@latest go.jetpack.io/typeid@latest || true\"",
```

Comment:
Aren't you:
a) Adding typeid and pkg to every project we have, whether they need it or not?
b) At this point, shouldn't we just do `go mod tidy` in all the projects?

Reply:
a) yes, but go-mod-sync runs `go mod tidy` which removes unused dependencies.
b) see (a)